src/main/content/_assets/js/instance-common.js

```diff
@@ -146,8 +146,8 @@ let InstancePane = class {
         let copyImgWrapper = $("<div/>", {class: "input-group-append"});
         copyImgWrapper.append(img);
 
-        let input = $("<input/>", {id, type: "text", class: "form-control collection-hub-input tooltip-copy", readonly: "readonly", onClick: "this.select();", value: url}) 
-            .tooltip({title: url, container: "body", placement: "top", trigger: "hover"});
+        let input = $("<input/>", {id, type: "text", class: "form-control collection-hub-input tooltip-copy", readonly: "readonly", onClick: "this.select();", value: "Copied!"}) 
```

Comment:
`value` attribute of the input that says "Copied!" doesn't make sense, it should be the URL of appsody or codewind.

Reply:
also the title should be the URL as well in the tooltip so when you hover over the input it shows the whole URL